src/SelectPanel/SelectPanel.stories.tsx

```diff
@@ -0,0 +1,88 @@
+import {Meta} from '@storybook/react'
+import React, {useState} from 'react'
+import {theme, ThemeProvider} from '..'
+import {Button} from '../Button'
+import {TriangleDownIcon} from '@primer/octicons-react'
+import {ItemInput} from '../deprecated/ActionList/List'
+import BaseStyles from '../BaseStyles'
+import {SelectPanel} from '../SelectPanel'
+import Box from '../Box'
+
+const meta: Meta = {
+  title: 'Components/SelectPanel',
+  component: SelectPanel,
+  decorators: [
+    (Story: React.ComponentType<React.PropsWithChildren<unknown>>): JSX.Element => {
+      return (
+        <ThemeProvider theme={theme}>
+          <BaseStyles>
+            <Story />
+          </BaseStyles>
+        </ThemeProvider>
+      )
+    },
+  ],
+  parameters: {
+    controls: {
+      disable: true,
+    },
+  },
+}
+export default meta
```

Comment:
Could we follow the same pattern as [previous](https://github.com/primer/react/pull/3195/files#diff-fe7e9d43d81744bd3e91c85e586cbb4b27e1eeb5f9e07211556a9eccd4205ab1R11)?

```
export default {
  title: 'Components/SelectPanel',
  component: SelectPanel,
  .........
} as ComponentMeta<typeof SelectPanel>
```